ntfysh_client/MainForm.cs

```diff
@@ -409,5 +411,32 @@ private void exitToolStripMenuItem1_Click(object sender, EventArgs e)
             _trueExit = true;
             Close();
         }
+
+        private async void SendNotificationMenuItem_Click(object sender, System.EventArgs e)
+        {
+            await SendMessage();
+        }
+
+        private async void notificationTopics_MouseDoubleClick(object sender, MouseEventArgs e)
+        {
+            await SendMessage();
+        }
+
+        private async Task SendMessage()
+        {
+            if (notificationTopics.SelectedItem != null && string.IsNullOrEmpty(notificationTopics.SelectedItem as string) == false)
+            {
+                var dlg = new SendMessageForm();
+                var r = dlg.ShowDialog(this);
+                if (r == DialogResult.OK)
+                {
+                    var topicAndHost = notificationTopics.SelectedItem.ToString()?.Split("@");
+                    var topic = topicAndHost[0];
+                    var host = topicAndHost[1].Replace("wss", "https");
+                    var msg = dlg.Message;
+                    await _notificationListener.SendNotification(host, new NtfyEvent { Topic = topic, Title = dlg.Title, Message = msg });
```

Comment:
Please handle the return value and return an error dialog box if we don't get a success response